src/drivers/serac.cpp

```diff
@@ -81,26 +78,15 @@ int main(int argc, char* argv[])
   // Save input values to file
   datastore.getRoot()->save("serac_input.json", "json");
 
-  // serial and parallel refinement levels
-  int ser_ref_levels;
-  inlet->get("ser_ref_levels", ser_ref_levels);  // has default value
-  int par_ref_levels;
-  inlet->get("par_ref_levels", par_ref_levels);  // has default value
-
-  // Build mesh
-  std::string mesh_file_path;
-  inlet->get("mesh", mesh_file_path);  // required in input file
-  mesh_file_path = serac::input::findMeshFilePath(mesh_file_path, input_file_path);
-  auto mesh      = serac::buildMeshFromFile(mesh_file_path, ser_ref_levels, par_ref_levels);
+  // Annotate the mesh with the input filename
+  inlet->addTable("nonlinear_solid/mesh_info")->addString("input_file_path")->defaultValue(input_file_path);
```

Comment:
I think the file path belongs at the root level instead of the `nonlinear_solid/mesh_info level`.

Reply:
For encapsulation reasons Inlet currently only passes the table corresponding to the mesh object (in this case, `/nonlinear_solid/mesh_info`) instead of the global table, so that subtable needs to contain all the information required to construct a mesh.  One option that Chris and I discussed is adding unrestricted access to global variables from subtables.

Reply:
I think that makes sense